packages/@aws-cdk/aws-ivs-alpha/lib/channel.ts

```diff
@@ -162,11 +206,22 @@ export class Channel extends ChannelBase {
       throw new Error(`channelName must contain only numbers, letters, hyphens and underscores, got: '${this.physicalName}'`);
     }
 
+    if ((props.type === ChannelType.STANDARD || props.type === ChannelType.BASIC) && props.preset !== undefined) {
+      throw new Error('preset cannot be used when STANDARD or BASIC channel type');
+    }
+
+    let preset = undefined;
+
+    if (props.type === ChannelType.ADVANCED_HD || props.type === ChannelType.ADVANCED_SD) {
+      preset = props.preset || Preset.HIGHER_BANDWIDTH_DELIVERY;
+    }
+
     const resource = new CfnChannel(this, 'Resource', {
       authorized: props.authorized,
       latencyMode: props.latencyMode,
       name: this.physicalName,
       type: props.type,
+      preset: preset,
```

Comment:
You can use the shorthand property name here:

```suggestion
      preset,
```